src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -170,7 +173,8 @@ def run(self):
                 self.cv.wait(timeout=5)
 
     def shutdown(self):
-        self.stopping.set()
+        self.stopping = True
```

Comment:
This is called by module shutdown (which is not even used anymore) but, despite that, the `shutdown` method **would be** called from a separate thread. (I mean, that's the entire point of a stopping flag. Why would you need it if it's only set by the thread itself?)

It's entirely conventional for a stopping flag on a thread to be a `threading.Event`. I really don't understand the motivation to change this.

Reply:
> It's entirely conventional for a stopping flag on a thread to be a threading.Event.

It is conventional to have lock on a boolean value when multiple threads will attempt to modify the boolean value. But since there's only one thread in this case, a boolean value without a lock is equally fine and therefore usage of `threading.Event` is unnecessary.

Reply:
Sorry, no. Anyone coming in the future that wants to stop threads now must play roulette with the cache lines hoping that the `shutdown` method  actually does what you would think in a properly multi-threaded class.

I urge you to consider this to be defensive programming and let the matter rest. There is no real performance gain to squeeze from this.

Reply:
> I urge you to consider this to be defensive programming

Defensive programming from the POV of possible future attempt to extend use of `self.stopping` is a good point.

> There is no real performance gain to squeeze from this.

Performance was never the main concern for this PR.